app/src/androidTest/java/com/github/sdpcoachme/MockTestRunner.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.sdpcoachme
+
+import android.app.Application
+import android.content.Context
+import androidx.test.runner.AndroidJUnitRunner
+
+class MockTestRunner : AndroidJUnitRunner() {
+    override fun newApplication(cl: ClassLoader?, className: String?, context: Context?): Application {
+        return super.newApplication(cl, CoachMeTestApplication::class.java.name, context)
+    }
+}
```

Comment:
Not entirely clear how that is going to be used and what it is for, maybe a comment describing the class would be good.